specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/componentProactiveDetection_API.json

```diff
@@ -0,0 +1,315 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ConfigurationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The ProactiveDetection configuration for this configuration id.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentProactiveDetectionConfiguration"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ProactiveDetectionConfigurationGet": {
+            "$ref": "./examples/ProactiveDetectionConfigurationGet.json"
+          }
+        }
+      },
+      "put": {
+        "description": "Update the ProactiveDetection configuration for this configuration id.",
+        "operationId": "ProactiveDetectionConfigurations_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ConfigurationId"
+          },
+          {
+            "name": "ProactiveDetectionProperties",
+            "description": "Properties that need to be specified to update the ProactiveDetection configuration.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentProactiveDetectionConfiguration"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The ProactiveDetection configuration that was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentProactiveDetectionConfiguration"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ProactiveDetectionConfigurationUpdate": {
+            "$ref": "./examples/ProactiveDetectionConfigurationUpdate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationInsightsComponentProactiveDetectionConfigurationListResult": {
+      "description": "A list of ProactiveDetection configurations.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/ApplicationInsightsComponentProactiveDetectionConfiguration"
+      }
+    },
+    "ApplicationInsightsComponentProactiveDetectionConfiguration": {
+      "description": "A ProactiveDetection configuration definition.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
```

Comment:
"tags" ARM top level property missing? Is this a tracked or proxy resource? Seems tracked since it has a location property